src/libs/OptionsListUtils.js

```diff
@@ -77,6 +78,45 @@ Onyx.connect({
     },
 });
 
+const policyExpenseReports = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.REPORT,
+    callback: (report, key) => {
+        if (!ReportUtils.isPolicyExpenseChat(report)) {
+            return;
+        }
+        policyExpenseReports[key] = report;
+    },
+});
+
+/**
+ * Get the options for a policy expense report.
+ * @param {Object} report
+ * @returns {Array}
+ */
+function getPolicyExpenseReportOptions(report) {
+    if (!ReportUtils.isPolicyExpenseChat(report)) {
+        return [];
+    }
+    const filteredPolicyExpenseReports = _.filter(policyExpenseReports, policyExpenseReport => policyExpenseReport.policyID === report.policyID);
+    return _.map(filteredPolicyExpenseReports, (expenseReport) => {
+        const policyExpenseChatAvatarSource = lodashGet(policies, [
+            `${ONYXKEYS.COLLECTION.POLICY}${expenseReport.policyID}`, 'avatar',
+        ]) || ReportUtils.getDefaultWorkspaceAvatar(expenseReport.displayName);
+        return {
+            ...expenseReport,
+            keyForList: expenseReport.policyID,
+            text: expenseReport.displayName,
```

Comment:
Not exactly a regression but we shouldn't have used `expenseReport.displayName` here because the display name is not necessary the policy name. [The display name is the policy name only if you are the policy expense chat owner](https://github.com/Expensify/App/blob/9754a32f38b6e05044aa5198d5be4b8b0b6b920b/src/libs/ReportUtils.js#L798-L801). 

This worked fine with money request because you are the only one who can see the participants list (and that's only at the creation phase). But that's not the case with split action.